Makefile

```diff
@@ -486,7 +486,7 @@ build-java-docker-dev:
 build-go-docker-dev:
 	docker buildx build --build-arg VERSION=dev \
 		-t feastdev/feature-server-go:dev \
-		-f go/infra/docker/feature-server/Dockerfile --load .
+		-f SCYLLADB_SHARD_AWARE_ENABLED=$(SCYLLADB_SHARD_AWARE_ENABLED) go/infra/docker/feature-server/Dockerfile --load .
```

Comment:
Why do we need this? Doesn't the drivers auto detect this?

Reply:
This is for conditionally selecting the the ScyllaDB fork of the `gocql` driver library which among other things has the feature to select the correct shard for ScyllaDB faster; we still don't use this, but the setup is here.

Reply:
https://opensource.docs.scylladb.com/stable/using-scylla/drivers/cql-drivers/scylla-go-driver.html

Reply:
Deleted, we should only be using the scylladb gocql fork